test/endpoints/googlenow/googlenow_test.go

```diff
@@ -323,7 +328,7 @@ func TestAddCredentialsMissingAuthCode(t *testing.T) {
 	}
 
 	assert.Equal(t, "202 Accepted", res2.Status)
-	assert.Equal(t, 1, counter)
+	assert.Equal(t, 2, counter)
```

Comment:
Earlier, the code would never hit the backend (it would return 400 from the required field missing and hence the counter would not get incremented). In the new case, we actually call the mock backend implemented above in the test - which goes results in the counter being incremented.